### DISCUSSION TITLE
Beatmap permission issues after a fresh install of Windows on a new drive

### DISCUSSION THREAD

**Discussion by RatinFX** [ppy/osu#25791] [Q&A]
### Type

Other

### Bug description

- fresh Windows11 install on a new drive,
- also reinstalled osu!lazer on it and changed the osu! folder to the already existing one on drive `D:\` in-game,
- it cannot read or import _some_ maps correctly unless it's run as an administrator - though doing so breaks the beatmap drag & drop import but that's a different issue

It might not entirely be a lazer-related issue but I would like to find the solution for it regardless

### Screenshots or videos

- beatmap in the video: https://osu.ppy.sh/beatmapsets/907927#osu/1894676

https://github.com/ppy/osu/assets/38523955/e031cca1-53c6-4265-9f5f-10cba3bebe50

### Version

2023.1130.0

### Logs

[runtime.log](https://github.com/ppy/osu/files/13692508/runtime.log)

**Comment by peppy** [comment]
The moment you run a game or app as administrator you're basically fucked. You'll need to fix your windows permissions on files which were set to admin owned.

**Reply by RatinFX** [reply]
That's a bit weird as it was never started lazer as admin before.
I tried setting the perms to the currently logged-in user on all lazer related folders with "Replace owner on subcontainers and objects" checked, yet nothing seemed to change, was that not what you meant?

Edit: I think it's good now, I also checked the "Replace all child object permission entries..." option below it as I noticed some didn't update correctly, thank you for the info o7
